### PR TITLE
add prerender bot

### PR DESCRIPTION
**Pull request by LinusGeffarth**
Not sure this is how it's supposed to be done, but I tried 😄

### CODE REVIEW

**Comment by LinusGeffarth** [comment]
Apparently this really isn't how to. @JayBizzle can you assist?

File: src/Fixtures/Crawlers.php [hunk 1]
```diff
@@ -904,6 +904,7 @@ class Crawlers extends AbstractProvider
         'postrank',
         'PowerPoint\/',
         'Prebid',
+        'Prerender'
```
Comment (JayBizzle) [inline src/Fixtures/Crawlers.php:907]: You missed a comma here

**Review by JayBizzle — CHANGES_REQUESTED** [review]

File: src/Fixtures/Crawlers.php (hunk 1 above)
Comment (LinusGeffarth) [reply src/Fixtures/Crawlers.php]: argh – fixed that

**Review by JayBizzle — APPROVED** [review]

**Comment by JayBizzle** [comment]
Thanks!

**Pull request merged**